documentation-site/components/yard/ast.ts

```diff
@@ -193,7 +193,7 @@ export function parseCode(code: string, elementName: string) {
             propValues[name] = value;
           });
           propValues['children'] = generate(
-            (path.node as any).children[0],
```

Comment:
@tajo I'm not aware of why this was restricted to first element. I needed to make this change so that all button elements were parsed

Reply:
not sure, it seems to work fine for all children (arrays) too